pandas-stubs/_libs/tslibs/offsets.pyi

```diff
@@ -125,7 +128,13 @@ class BusinessMixin(SingleConstructorOffset):
         self, n: int = ..., normalize: bool = ..., offset: timedelta = ...
     ) -> None: ...
 
-class BusinessDay(BusinessMixin): ...
+class _NotForPeriodBusinessMixin(_NotForPeriodSingleConstructorOffset):
```

Comment:
Those two private helper classes seem to be only used by BusinessDay. Would it make sense to add their methods to BusinessDay instead?

Reply:
Good point.  I got rid of the two classes, but made a comment in there that indicates why it is done this way.